src/metabase/util/schema.clj

```diff
@@ -501,9 +507,10 @@
   "Schema for a string that is a valid representation of a boolean (either `true` or `false`).
   Something that adheres to this schema is guaranteed to to work with `Boolean/parseBoolean`."
   (mc/schema
-    [:and {:error/fn (constantly (deferred-tru "value must be a valid boolean string (''true'' or ''false'')."))}
+    [:and
```

Comment:
what do you think of using `[:enum "true" "false"]` here? you did setup those tests to make sure the new m.schemas are equal to the old ones, but if it's too drastic a change that makes sense too